### PR TITLE
refactorizacion_de_codigo

### PR DESCRIPTION
**Pull request by ErickaIsa**
se refactorizo mi codigo

### CODE REVIEW

File: api_endpoints/mail_important_endpoints.py
Comment (AlisonGuzman2006) [inline api_endpoints/mail_important_endpoints.py:1]: The changes have been reviewed and accepted

File: config/config.py
Comment (AlisonGuzman2006) [inline config/config.py:1]: The addition of the new code is approved

File: payloads/mail_important/correo_image_importan.json
Comment (AlisonGuzman2006) [inline payloads/mail_important/correo_image_importan.json:1]: The addition of the new code has been reviewed and approved

File: resources/auth/auth.py
Comment (AlisonGuzman2006) [inline resources/auth/auth.py:1]: The changes have been reviewed and accepted

File: payloads/mail_important/mail_important.json
Comment (AlisonGuzman2006) [inline payloads/mail_important/mail_important.json:1]: The addition of the new code has been reviewed and accepted

File: tests/mail_important/conftest.py
Comment (AlisonGuzman2006) [inline tests/mail_important/conftest.py:1]: The addition of the new code has been reviewed and accepted

File: tests/mail_important/test_mail_important.py
Comment (AlisonGuzman2006) [inline tests/mail_important/test_mail_important.py:1]: The changes have been reviewed and accepted

**Review by AlisonGuzman2006 — APPROVED** [review]
The changes and new code have been reviewed and approved

**Review by Jasonn5 — APPROVED** [review]
executed successfully, good job

**Pull request merged**